virttest/utils_sys.py

```
@@ -191,3 +191,23 @@ def get_qemu_log(vms, type="local", params=None, log_lines=10):
+    :param test: test object.
+    """
+    log_path = params.get("libvirtd_debug_file")
+    remote_session = params.get("remote_session")
+    remote_file_type = params.get("remote_file_type", "virtqemud")
+    log_lines = params.get("log_lines", 50)
+    try:
+        cmd = f"tail -n {log_lines} {log_path}"
+        remote_output = remote_session.cmd_output(cmd, timeout=10).strip()
+        test.log.debug(f"=== Print {remote_file_type} log ===\n{remote_output}")
+
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)